### ISSUE TITLE
Generator logic fails for src/config.js

### ISSUE THREAD

**Issue by dustingraves** [prescottprue/generator-react-firebase#84]
**Do you want to request a *feature* or report a *bug*?**  
Bug

**What is the current behavior?**
The output of the generator in `src/config.js` from [https://github.com/prescottprue/generator-react-firebase/blob/master/generators/app/templates/src/config.js#L24](line 24)
prints `truefalse` if you select redux.

**What is the expected behavior?**
It should print true or false

**Which versions of dependencies, and which browser and OS are affected by this issue? Did this work in previous versions or setups?**

This is from master branch without any changes.

**Comment by dustingraves** [comment]
```javascript
export const reduxFirebase = {
  userProfile: 'users', // root that user profiles are written to
  enableLogging: false, // enable/disable Firebase Database Logging
  useFirestoreForProfile: truefalse, // Save profile to Firestore instead of Real Time Database
  // updateProfileOnLogin: false // enable/disable updating of profile on login
  // profileDecorator: (userData) => ({ email: userData.email }) // customize format of user profile
}
```

**Comment by prescottprue** [comment]
@dustingraves did you select to use Firestore or no?

**Comment by dustingraves** [comment]
@prescottprue I did select to use firestore

**Comment by prescottprue** [comment]
Fixed in [`v2.0.0-rc.3`](https://github.com/prescottprue/generator-react-firebase/releases/tag/v2.0.0-rc.3). Thanks for reporting.
